lab_1/main.py

```diff
@@ -3,75 +3,70 @@
 A concordance extraction
 """
 
-
+import re
 def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-text = text.split ()
-return text
-
+    pass
+    if isinstance(text, str):
+        text = text.lower()
+        text = re.sub(r'[^a-zA-Z0-9\s ]', '', text)
+        text = text.split()
+        return text
+    else:
```

Comment:
you don't need else here (after if with return)
if return text works, the function will finish its work